src/content/docs/en/guides/client-side-routing.mdx

```diff
@@ -0,0 +1,192 @@
+---
+title: Client-side routing (experimental)
+description: >-
+  How to enable experimental support for client-side routing in your Astro site.
+i18nReady: true
+---
+
+Support for client-side routing in Astro projects can be enabled behind an experimental flag. Client-side routing in Astro is powered by a new browser API, [view transitions](https://developer.chrome.com/docs/web-platform/view-transitions/) with fallback options for non-supporting browsers.
```

Comment:
I feel we're being a bit redundant here, maybe reword it like this?

Also, as we're naming an API, I suppose it should be treated as a proper noun?

```suggestion
Support for client-side routing in Astro projects can be enabled behind an experimental flag. It is powered by the new [View Transitions](https://developer.chrome.com/docs/web-platform/view-transitions/) browser API, with fallback options for non-supporting browsers.
```